packages/cordova-plugin/dist/definitions.d.ts

```diff
@@ -0,0 +1,130 @@
+export type PluginError = {
+    code: string;
+    message: string;
+};
+export type OSGLOCPosition = {
+    timestamp: number;
+    latitude: number;
+    longitude: number;
+    accuracy: number;
+    altitude: number | null;
+    speed: number | null;
+    heading: number | null;
+    altitudeAccuracy: number | null;
+};
+export type LegacyOSPosition = {
+    timestamp: number;
+    latitude: number;
+    longitude: number;
+    accuracy: number;
+    altitudeAccuracy: number | null | undefined;
+    altitude: number | null;
+    velocity: number | null;
+    heading: number | null;
+};
```

Comment:
Do we need both? They are identical so I was wondering if we could reuse one of them.